src/main.rs

```diff
@@ -182,6 +182,12 @@ fn main() {
         .build()
     {
         Ok(re) => walk::scan(&dir_vec, Arc::new(re), Arc::new(config)),
-        Err(err) => error(err.description()),
+        Err(err) => error(
+            format!(
+                "{} \n(Hint: Try the --fixed-strings \
```

Comment:
I guess the space after `{}` can be removed.

Reply:
@sharkdp I will remove this useless space :)